examples/notebooks/tour-of-beam/getting-started.ipynb

```diff
@@ -0,0 +1,1020 @@
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "SVk4BsuKVlLj",
+        "colab_type": "text"
+      },
+      "source": [
+        "Now let's dive into creating our first pipeline.\n",
+        "\n",
+        "For this first pipeline, let's just feed it some data from a Python list and print the results.\n",
+        "\n",
+        "Each _step_ in the pipeline is delimited by the _pipe operator_ `|`.\n",
+        "The outputs of each transform are passed to the next transform as inputs.\n",
+        "And we can save the final results into a `PCollection` variable.\n",
+        "\n",
+        "```py\n",
+        "# We pass the elements from step1 through step3 and save the results into `outputs`.\n",
+        "outputs = pipeline | step1 | step2 | step3\n",
+        "```\n",
+        "\n",
+        "Pipelines can quickly grow long, so it's sometimes easier to read if we surround them with parentheses and break them into multiple lines.\n",
+        "\n",
+        "```py\n",
+        "# This is equivalent to the example above.\n",
+        "outputs = (\n",
+        "  pipeline\n",
+        "  | step1\n",
+        "  | step2\n",
+        "  | step3\n",
+        ")\n",
+        "```\n",
+        "\n",
+        "Also, Beam expects each transform, or step, to have a unique _label_, or description.\n",
+        "This makes it a lot easier to debug, and it's in general a good practice to start.\n",
+        "You can use the _right shift operator_ `>>` to add a label to your transforms, like `'My description' >> MyTransform`.\n",
+        "\n",
+        "```py\n",
+        "# Try to give short but descriptive labels.\n",
+        "# These serve both as comments and help debug later on.\n",
+        "outputs = (\n",
+        "  pipeline\n",
+        "  | 'First step' >> step1\n",
+        "  | 'Second step' >> step2\n",
+        "  | 'Third step' >> step3\n",
+        ")\n",
+        "```\n",
+        "\n",
+        "> ℹ️&nbsp; The syntax might seem a little strange at first, but you'll become familiar with it.\n",
```

Comment:
How about "a little different"?

We've heard that the use of custom operators is confusing at first for Python users since those operators are not commonly used, and I wanted to empathize with those users. If you have a better suggestion please let me know, I wasn't too sure on using "strange" to begin with :)

Reply:
"a little different" - SGTM.